### DISCUSSION TITLE
Web GUI using only Z80pack software

### DISCUSSION THREAD

**Discussion by Zhan85** [thehighnibble/thehighnibble.github.io#184] [IMSAI8080esp replica]
Curious.  Is there anyway to access the WEB GUI desktop mode using just the latest Z80pack on a Linux machine?  Also wondering if I could just flash any ESP32 with the Imsai Sim firmware and be able to access all the functions over the webgui?

**Comment by udo-munk** [comment]
Yes on Linux systems you can access the WEB GUI, to enable run machines with option -n.
Don't know if one could just flash some ESP32 board and have it working, Dave will know this.
